homeassistant/components/knx/translations/en.json

```
@@ -51,13 +51,13 @@
                     "multicast_group": "Multicast group used for routing and discovery",
                     "multicast_port": "Multicast port used for routing and discovery",
                     "rate_limit": "Maximum outgoing telegrams per second",
-                    "state_updater": "Globally enable reading states from the KNX Bus"
+                    "state_updater": "Globally enable reading states from the KNX Bus",
+                    "threaded": "EXPERIMENTAL: \n Use an extra thread for the KNX communication"
```

```suggestion
                    "threaded": "EXPERIMENTAL: Use an extra thread for the KNX communication"
```
not sure if this auto-generates from `strings.json` or if we should fix it 🤔